### DISCUSSION TITLE
Add status toggle slider

### DISCUSSION THREAD

**Discussion by thuongtruong1009** [CodingContributorsLair/BMICalculator#22] [Q&A]
After following your demo product, I have also built such a product, so I have a few suggestions as follows:
- I added a slider that shows the state of the body depending on the BMI. I took its length and divided it by 100%. Similarly, I assign BMI from 0-50. As a result, 1% of the sider bar length = 2 BMI. And in the Javascript part I take that number as margin-left. So, when BMI is calculated, the slider pointer will slide right 1 space equal to margin-left. It is similar to the following figure:
## If you found my comment helpful, please mark it as the answer that helped me

![image](https://user-images.githubusercontent.com/71834167/133871321-e73b26bb-0c0f-4f97-a5ab-f6f245251b0f.png)
## If you found my comment helpful, please mark it as the answer that helped me. That will give more people the opportunity to contribute and learn
![image](https://user-images.githubusercontent.com/71834167/133871390-be9bb6c8-8505-4626-b967-8b26e24f2783.png)

**Comment by NS007-dev** [comment]
@thuongtruong1009 so this slider will automatically move, showing the corresponding answer to the BMI after the BMI is calculated? That's clever and really cool! 

I can't seem mark it as an answer, it seems to only show this:

![image](https://user-images.githubusercontent.com/70807500/133880943-1a659d55-3238-45ad-a5bb-c422eca4b4fb.png)

Thank you so much!

**Reply by thuongtruong1009** [reply]
> @thuongtruong1009 so this slider will automatically move, showing the corresponding answer to the BMI after the BMI is calculated? That's clever and really cool!
> 
> I can't seem mark it as an answer, it seems to only show this:
> 
> ![image](https://user-images.githubusercontent.com/70807500/133880943-1a659d55-3238-45ad-a5bb-c422eca4b4fb.png)
> 
> Thank you so much!

This is there (below this comment). Mark it will help me pin on your discuss
![image](https://user-images.githubusercontent.com/71834167/133881067-506e8ea1-a092-406d-8768-14c46f014710.png)
--> You also should add a transition in CSS effect to smooth them and add auto change number on toogle index.
Thank you very much💖
